code/modules/mob/living/carbon/human/human.dm

```diff
@@ -528,12 +528,13 @@
 
 //Used for new human mobs created by cloning/goleming/podding
 /mob/living/carbon/human/proc/set_cloned_appearance()
-	if(gender == MALE)
+	if(dna.features["body_model"] == MALE)
```

Comment:
This is horribly ugly, is there no variable that stores the body model on the human?

Reply:
> This is horribly ugly, is there no variable that stores the body model on the human?

I had to find a way to make it work with cloning, the solution I found was the citadel PR which puts it as a feature of the mob.

Reply:
Also think that moving it out of the features would make some trouble down the road as changelings would probably be easily identifiable if someone were to strip them and they didn’t look like how the original did. 

Also believe that I experimented with other possible methods of making this work through the TG body type PR which wasn’t compatible with cloning and my attempts at making it work all resulted in failures.